app/views/foods/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= link_to "Add New Food", new_food_path, style: "width: fit-content;" , class: "btn btn-primary" %>
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Unit</th>
+      <th>Price</th>
+      <th>Action</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @foods.each do |food| %>
+      <tr>
+        <td><%= food.name %></td>
+        <td><%= food.measurement_unit %></td>
+        <td><%= food.price %></td>
+        <td>
+          <%= link_to "Delete", food, data: { turbo_method: :delete, confirm: "Are you sure?" } %>
```

Comment:
- [ ] When I try to delete a food, I am getting this error bellow, I think it's happening because we want to delete a book that does not belong to the current user logedin, kindly fix this and make sure the dellet button only appears for the user owner of the food and he should be the only one who can delete it.
![image](https://github.com/jlvFlores/RoR-recipe-app/assets/65068771/d59acdcc-75c8-4cd0-b712-3a33cfbffeb6)